api/openapi/activities.yaml

```diff
@@ -0,0 +1,234 @@
+          description: The offset used for this page of results
+          required: true
+        - schema:
+            type: string
+          in: query
+          name: sort
+          description: Indicate sort by field
+          required: true
+        - schema:
+            type: string
+            enum:
+              - ASC
+              - DESC
+          in: query
+          name: order
+          description: Indicate sorting direction
+          required: true
+      requestBody:
+        $ref: '#/components/requestBodies/ActivitySearchDto'
+components:
+  schemas:
+    Activity:
+      title: Activity
+      x-stoplight:
+        id: o6vlu07vubno4
+      type: object
+      description: activity details
+      properties:
+        id:
+          type: integer
+          x-stoplight:
+            id: n5nyuo08bl1gy
+          format: int64
+          minimum: 1
+        created_at:
+          type: string
+          x-stoplight:
+            id: 3ftqmjgnnuxw7
+          format: date-time
+        action:
+          type: string
+          x-stoplight:
+            id: u55lof2lfi49k
+        event_name:
+          type: string
+          x-stoplight:
+            id: nbqeh7nrhgbn9
+        priority:
+          type: string
+          x-stoplight:
+            id: biefyfwvci7lw
+        object_id:
+          type: integer
+          x-stoplight:
+            id: kd05o5bwbg6wl
+          exclusiveMinimum: false
+          exclusiveMaximum: false
+          format: int64
+          minimum: 1
+        object_name:
+          type: string
+          x-stoplight:
+            id: pxcosifywpqkn
+        object_type:
+          type: string
+          x-stoplight:
+            id: bl5qkiasycg5i
+        project_id:
+          type: integer
+          x-stoplight:
+            id: byvwz3x6nsyaa
+          format: int64
+          exclusiveMinimum: false
+          minimum: 1
+        project_name:
+          type: string
+          x-stoplight:
+            id: v3ro2mr4suog9
+        subject_name:
+          type: string
+          x-stoplight:
+            id: unf6lpxufh2xk
+        subject_type:
+          type: string
+          x-stoplight:
+            id: y1hoik59fp0ie
+        details:
+          type: object
+          x-stoplight:
+            id: eygyh7jp5k2wz
+          properties:
+            history:
+              x-stoplight:
+                id: 9f0iaa69imgaj
+              type: array
+              items:
+                $ref: '#/components/schemas/HistoryField'
+    HistoryField:
+      title: HistoryField
+      x-stoplight:
+        id: zn71tf8rb2xfu
+      type: object
+      properties:
+        field:
+          type: string
+          x-stoplight:
+            id: kr5ju3d2ftb61
+        old_value:
+          type: string
+          x-stoplight:
+            id: 9zzt3diczvwnc
+        new_value:
+          type: string
+          x-stoplight:
+            id: gcydbgmi7or7q
+    SearchCriteria:
```

Comment:
@rkukharenka shall we divide it into a sharing model?

Reply:
I think we can move this to a sharing model if the need arises in the future